Source/NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationBallClipRotate.swift

```diff
@@ -32,22 +32,16 @@ class NVActivityIndicatorAnimationBallClipRotate: NVActivityIndicatorAnimationDe
     func setUpAnimation(in layer: CALayer, size: CGSize, color: UIColor) {
         let duration: CFTimeInterval = 0.75
 
-        //    Scale animation
-        let scaleAnimation = CAKeyframeAnimation(keyPath: "transform.scale")
-
-        scaleAnimation.keyTimes = [0, 0.5, 1]
-        scaleAnimation.values = [1, 0.6, 1]
-
         // Rotate animation
         let rotateAnimation = CAKeyframeAnimation(keyPath: "transform.rotation.z")
 
-        rotateAnimation.keyTimes = scaleAnimation.keyTimes
+        rotateAnimation.keyTimes = [0, 0.5, 1]
```

Comment:
I have applied here the same changes Nico made in these commits: [commit](https://github.com/mozioinc/NVActivityIndicatorView/commit/55233cd249c6850f501637c8bcf3d04202ac3241) & [commit](https://github.com/mozioinc/NVActivityIndicatorView/commit/2b6c141530aa6a21169c2b9a2c8b2dfd54fbea8a).
The class was renamed so Git didn't detect the changes